tests/django/test_vulnerable.py

```diff
@@ -104,3 +104,33 @@ def test_deserialization_yaml_load_all_normal(client, method_name):
         },
     )
     assert response.status_code == 200
+
+
+@pytest.mark.parametrize("method_name", ["get", "post"])
+def test_xxe_lxml_etree_fromstring_normal(client, method_name):
```

Comment:
Not a big deal, but I think each of these could be parametrized on the endpoint since they all send the same `user_input`.

Reply:
all of that will be tackled [here](https://contrast.atlassian.net/browse/PYT-992)

Reply:
I agree, the unit tests are going to be overhauled into something that likely resembles the screener config, so I'm not too worried about what they look like for now (as long as we get that perfect coverage)